test/driver.js

```diff
@@ -687,6 +687,18 @@ class Driver {
               // like IRL (in order to avoid bugs like bug 1907958).
               annotation.quadPoints = new Float32Array(quadPoints);
             }
+            if (paths) {
+              for (let i = 0, ii = paths.lines.length; i < ii; i++) {
+                paths.lines[i] = new Float32Array(
+                  paths.lines[i].map(x => x ?? NaN)
+                );
```

Comment:
Can't this be "simplified" a tiny bit?
```suggestion
                paths.lines[i] = Float32Array.from(paths.lines[i], x => x ?? NaN);
```